susie/src/main/scala/MakeSuSiE.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dig.aggregator.methods.susie
+
+import org.broadinstitute.dig.aggregator.core._
+import org.broadinstitute.dig.aws.emr._
+import org.broadinstitute.dig.aws.Ec2.Strategy
+import org.broadinstitute.dig.aws.MemorySize
+
+class MakeSuSiE(implicit context: Context) extends Stage {
+  import MemorySize.Implicits._
+
+  val ancestrySpecific: Input.Source = Input.Source.Success("out/metaanalysis/bottom-line/ancestry-clumped/*/ancestry=EU/")
+
+  /** Source inputs. */
+  override val sources: Seq[Input.Source] = Seq(ancestrySpecific)
+
+  /** Map inputs to their outputs. */
+  override val rules: PartialFunction[Input, Outputs] = {
+    case ancestrySpecific(phenotype, ancestry) => Outputs.Named(s"$phenotype/${ancestry.split('=').last}")
```

Comment:
if you are going to specify ancestry=EU above this has to be changed since there is only one wildcard now:

```
case ancestrySpecific(phenotype) => Outputs.Named(s"$phenotype/EU")
```

will do it